### PR TITLE
add supported chains to chain fusion section.

### PR DESCRIPTION
**Pull request by domwoe**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [ ] Follows the [developer docs style guide](style-guide.md).
- [ ] Follows the [best practices and guidelines](README.md#best-practices).
- [ ] New documentation pages include [document tags](README.md#document-tags).
- [ ] New documentation pages include [SEO keywords](README#seo-keywords).
- [ ] New documents are in the `.mdx` file format to support the previous two components. 
- [ ] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+# Supported chains
+
+<MarkdownChipRow labels={["Advanced", "Chain Fusion"]} />
+
+## Overview
+
+A chain can be supported if it leverages an authentication method that is
+supported by the threshold signing protocols of the Internet Computer, and has
+reliable RPC providers supporting IPv6. For some chains, DFINITY or the
+community has built libraries or canisters to simplify the integration.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:50]:
```suggestion
Other blockchain networks can be integrated with ICP if they leverage an authentication method that is
supported by the threshold signing protocols of the Internet Computer, and has
reliable RPC providers supporting IPv6. 

For some chains, DFINITY or the ICP community has built libraries or canisters to simplify the integration.
```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+## Selection of supported chains
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:52]:
```suggestion
## Supported chains
```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+The following table lists a selection of supported chains and their respective
+authentication methods, available integrations, and chain-key tokens. EVM
+compatible chains other than Ethereum are not listed explicitly, but they are
+supported in the same way as Ethereum.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:57]:
```suggestion
The following table lists a selection of supported chains and their respective
authentication methods, available integrations, and chain-key tokens. 

:::caution
EVM-compatible chains other than Ethereum are not listed explicitly, but they are
supported in the same way as Ethereum through the EVM RPC canister.
:::
```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+- **Authentication**: Authentication methods, i.e. signature schemes used to
+  authenticate transactions, that are supported by the Internet Computer's
+  threshold signing capabilities.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:61]:
```suggestion
- **Authentication**: Authentication methods, such as signature schemes used to authenticate transactions, that are supported by ICP's threshold signing capabilities.
```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+- **Available Integrations**: The available integrations for each chain, i.e.
+  the methods by which the Internet Computer can interact with the chain.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:64]:
```suggestion
- **Available integrations**: The available integrations for each chain, i.e.
  the methods by which ICP can interact with the chain.
```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+| Filecoin  | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Stacks    | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Sui       | ECDSA, EdDSA                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Arweave   | [Not yet supported](https://github.com/ArweaveTeam/arweave/issues/486) | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Stellar   | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+
+:::note
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:92]:
```suggestion
:::caution
```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+Threshold Schnorr and EdDSA signing are not available in production yet.
+Threshold Schnorr signing will be released with the Deuterium milestone
+scheduled for August 2024. Threshold EdDSA signing will be released with the
+Helium milestone later this year. However, both signing schemes are already
+available as a [developer preview](https://github.com/domwoe/schnorr_canister).
+
+:::
+
+## The chain you want to integrate is not listed?
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:102]:
```suggestion
### Looking for a chain that is not listed?
```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+Most likely, the chain you want to integrate is still supported by the Internet
+Computer. Check the docs of the chain you want to integrate to find out which
+authentication/signature schemes are supported.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:106]:
```suggestion
Most likely, the chain you want to integrate is still supported by ICP. Check the docs of the chain you want to integrate to find out which authentication methods or signature schemes are supported.
```

**Comment by jessiemongeon1** [comment]
Is there a plan on how to make sure this list stays up to date with the latest authentication methods supported by each chain?  Has each chain that is listed been verified through a sample canister that an integration works as expected?

**Comment by domwoe** [comment]
@jessiemongeon1 It is highly unlikely that an available authentication method gets deprecated (except when quantum computers become a thread). We don't have sample canisters. I've verified based on the available documentation.

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+    Kaspa,
+    Aptos,
+    Monero,
+    Hedera,
+    Filecoin,
+    Stacks,
+    Sui,
+    Arweave,
+    Stellar,
+    Chain Fusion,
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:32]:
```suggestion

```

File: docs/developer-docs/multi-chain/supported-chains.mdx [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+| Filecoin  | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Stacks    | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Sui       | ECDSA, EdDSA                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Arweave   | [Not yet supported](https://github.com/ArweaveTeam/arweave/issues/486) | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
+| Stellar   | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/supported-chains.mdx:94]:
```suggestion

| Aptos     | [ECDSA, EdDSA](https://aptos.dev/en/network/blockchain/accounts)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |    
| Arweave   | [Not yet supported](https://github.com/ArweaveTeam/arweave/issues/486) | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Avalanche | [ECDSA](https://docs.avax.network/reference/standards/cryptographic-primitives)                                                                 | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Bitcoin   | [ECDSA, Schnorr](/docs/current/developer-docs/multi-chain/bitcoin/using-btc/generate-addresses)                                                        | Direct                                                                                                                                         | [ckBTC](docs/developer-docs/multi-chain/chain-key-tokens/ckbtc/overview.mdx)                                                                    |
| Cardano   | [ECDSA, EdDSA, Schnorr ](https://developers.cardano.org/docs/governance/cardano-improvement-proposals/cip-0049/)                                                 | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| Cosmos    | [ECDSA, EdDSA](https://docs.cosmos.network/v0.50/learn/beginner/accounts)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Dogecoin  | ECDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| Ethereum  | [ECDSA](/docs/current/developer-docs/multi-chain/ethereum/using-eth/generating-addresses)                                                                  | [EVM RPC](docs/developer-docs/multi-chain/ethereum/evm-rpc/overview.mdx), [ic-web3-rs](https://github.com/horizonx-tech/ic-web3-rs)            | [ckETH](docs/developer-docs/multi-chain/chain-key-tokens/cketh/overview.mdx), ckERC20                                                           |
| Filecoin  | [ECDSA](https://spec.filecoin.io/algorithms/crypto/signatures/)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Hedera    | [ECDSA, EdDSA](https://docs.hedera.com/hedera/core-concepts/keys-and-signatures)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Kaspa     | [ECDSA](https://github.com/kaspanet/go-secp256k1/blob/master/ecdsa_privkey.go)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Monero    | [EdDSA](https://getmonero.dev/cryptography/asymmetric/edwards25519.html)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| NEAR      | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Polkadot  | [ECDSA, EdDSA](https://wiki.polkadot.network/docs/learn-cryptography)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Solana    | [EdDSA](https://solana.com/docs/core/accounts#account)                                                                  | [Solana RPC](https://github.com/mfactory-lab/ic-solana) <span style={{ display: 'inline-block', verticalAlign: 'super' }}><AlphaChip /></span> | [gSOL](https://galactic-bridge-tzbjl.ondigitalocean.app/) <span style={{ display: 'inline-block', verticalAlign: 'super' }}><BetaChip /></span> |
| Stacks    | [ECDSA](https://github.com/stacks-network/keylib-py)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Stellar   | [EdDSA](https://developers.stellar.org/docs/smart-contracts/smart-wallets#secp256r1)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| Sui       | [ECDSA, EdDSA](https://blog.sui.io/wallet-cryptography-specifications/)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |
| XRP       | [ECDSA, EdDSA](https://xrpl.org/docs/concepts/accounts/cryptographic-keys/#signing-algorithms)                                                           | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| Toncoin   | EdDSA                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
| TRON      | [ECDSA](https://github.com/tronprotocol/tips/issues/120)                                                                  | [RPC via HTTPS Outcalls](docs/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview.mdx)                     |                                                                                                                                                 |
```

Comment (jessiemongeon1) [reply docs/developer-docs/multi-chain/supported-chains.mdx:94]: Reorganized by alphabetical order and added links to each chain's documentation that specifies its supported signatures

Comment (domwoe) [reply docs/developer-docs/multi-chain/supported-chains.mdx:94]: I'm not sure if this is a good idea as links may change, and there's not a huge value in having these links available.

**Pull request merged**